storage-bigtable/src/access_token.rs

```diff
@@ -91,41 +91,46 @@ impl AccessToken {
     }
 
     /// Call this function regularly to ensure the access token does not expire
-    pub async fn refresh(&self) {
+    pub fn refresh(&self) {
         // Check if it's time to try a token refresh
-        {
```

Comment:
I think it would not hurt to keep this block wrapper to make the drop of the read handle more explicit. Do you mind restoring it?

Reply:
sorry, forgot to run fmt
I added `drop(token_r);` :)